kaspresso/src/main/kotlin/com/kaspersky/kaspresso/instrumental/InstrumentalUtils.kt

```diff
@@ -0,0 +1,13 @@
+package com.kaspersky.kaspresso.instrumental
+
+import androidx.test.platform.app.InstrumentationRegistry
+
+/**
+ * @return true if an environment is instrumental, or false if the environment is JVM (with Robolectric)
+ */
+internal fun checkInstrumentalStateOfEnvironment(): Boolean =
```

Comment:
I actually prefer your approach to the one `AndroidJunit4.java` follow.

For documentation purposes, I leave it here as a comment, since this is how Google made Espresso tests compatible with robolectric:
```kotlin
private static String getRunnerClassName() {
    String runnerClassName = System.getProperty("android.junit.runner", null);
    if (runnerClassName == null) {
      if (!System.getProperty("java.runtime.name").toLowerCase().contains("android")
          && hasClass("org.robolectric.RobolectricTestRunner")) {
        return "org.robolectric.RobolectricTestRunner";
      } else {
        return "androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner";
      }
    }
    return runnerClassName;
  }
```

That's why in order to make test able to run on the JVM, one has to use `@RunWith(AndroidJunit4.class)`
You can findAndroidJunit4.class code [here](https://github.com/android/android-test/blob/master/ext/junit/java/androidx/test/ext/junit/runners/AndroidJUnit4.java)